templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/includes/new_public_participation_statement_confirm_content.html.twig

```diff
@@ -37,7 +37,11 @@
     {% block user_data_hint %}
         <p>
             {% if projectType == 'gateway' %}
-                {{ "explanation.data.user.change"|trans({ href: gatewayURL, label: 'gateway'|trans })|wysiwyg }}
+                {% if not hasPermission('feature_not_show_notice_comment_release_draft_statement') %}
+                    {{ "explanation.data.user.change"|trans({ href: gatewayURL, label: 'gateway'|trans })|wysiwyg }}
+                {% else %}
```

Comment:
i'm not sure you need this else here but this need a Frontend revview anyway

Reply:
done in [f6a0d1c](https://github.com/demos-europe/demosplan-core/pull/2721/commits/f6a0d1ca0a0f30aa3b1e35ff7eec9763449ccc11)